tests/test_invenio_circulation.py

```diff
@@ -19,16 +19,3 @@ def test_version():
     """Test version import."""
     from invenio_circulation import __version__
     assert __version__
-
-
-def test_init():
```

Comment:
Why did you remove this?

Reply:
because it needs other invenio extensions in order to be initialized so it was getting too much for just to check if is in extensions.